src/ZEO/ClientStorage.py

```diff
@@ -53,6 +53,9 @@
 
 logger = logging.getLogger(__name__)
 
+# max signed 64-bit value ~ infinity :) Signed cuz LBTree and TimeStamp
+m64 = b'\x7f\xff\xff\xff\xff\xff\xff\xff'
```

Comment:
> I expect [m64] will eventually migrate to ZODB.utils.

We also have such value (named MAX_TID) in NEO because of SQLite (which does not accept numbers above 2^63-1). So that can be a good idea to specify it in ZODB, but we would not use it in NEO, because we want to limit the ZODB dependency to the client (tha's why we also redefine a few things from ZODB.utils).